extension/public/manifest.json

```
@@ -3,23 +3,28 @@
     "description": "Temporary Extension description",
     "version": "1.0",
     "manifest_version": 3,
+    "content_scripts": [{
+        "matches": ["*://*/*"],
+        "js": ["contentScript.js"]
+    }],
     "icons": {
         "16": "icons/temp_icon_16.png",
         "32": "icons/temp_icon_32.png",
         "48": "icons/temp_icon_48.png",
         "128": "icons/temp_icon_128.png"
     },
     "background": {
-        "service_worker": "background.ts"
+        "service_worker": "background.js"
     },
     "permissions": [
-        "activeTab"
+        "activeTab",
+        "tabs"
     ],
     "host_permissions": [
-        "*://*.site.com/*"
+        "*://*/*"
     ],
     "action": {
         "default_icon": "icons/temp_icon_16.png",
         "default_popup": "popup.html"
     }
-}
+}
```

```suggestion
}

```
nit: add a new line at EOF